### ISSUE TITLE
nbcommands assumes python as the default language for notebooks

### ISSUE THREAD

**Issue by deepu-tp** [vinayak-mehta/nbcommands#7]
Notebooks store metadata about the kernel and language.
We could reuse this metadata for syntax highlighting, comments etc.

**Comment by vinayak-mehta** [comment]
Yes! Right now nbcommands just usels the Pygments' `PythonLexer` to highlight code, we could use the metadata to use other lexers as well.
